lib/configsetup.inc.php

```diff
@@ -227,6 +227,28 @@
             'name' => 'picture[preview_before_processing]',
             'value' => $config['picture']['preview_before_processing'],
         ],
+        'picture_retry_on_error' => [
```

Comment:
Keep in general section of better move to picture section?

Reply:
no, else we would have to add it to collage section too and also handle it there differen